docs/SPECIFICATION.md

```diff
@@ -234,9 +234,9 @@ A UserDataAddBody in a Message `m` is valid only if it passes these validations:
 8. If `m.data.body.type` is `USER_DATA_TYPE_USERNAME`, value must map to a valid fname.
 9. `m.data.body.value` must be a valid utf-8 string
 
-An fname is considered valid only if the most recent event for the fid `Transfer` event with the custody address in the `to` property. If a valid fname for a given fid becomes invalid, and there is a UserDataAdd message for that fid with the fname as its value, it must be revoked.
+A username is considered valid only if the most recent event for the fid `Transfer` event with the custody address in the `to` property. If a valid fname for a given fid becomes invalid, and there is a UserDataAdd message for that fid with the fname as its value, it must be revoked.
```

Comment:
we might want to specify invalidation rules for checking when ens names are expired (i.e., should check 1x per day)